cpp/src/io/orc/stripe_data.cu

```diff
@@ -1556,7 +1556,7 @@ __global__ void __launch_bounds__(block_size)
           if (t == 0) { s->top.data.buffered_count = n; }
         }
 
-        numvals = min(numvals * 8, is_last_set ? s->top.data.max_vals : blockDim.x);
+        numvals = min(numvals * 8, is_last_set ? (s->top.data.max_vals + 7) & (~0x7) : blockDim.x);
```

Comment:
Given how common this is, we should have a ceil to next byte function rather than have a +7)&~7 everywhere.